docs/Manifest.toml

```diff
@@ -0,0 +1,101 @@
+[[Base64]]
+uuid = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
+
+[[Dates]]
+deps = ["Printf"]
+uuid = "ade2ca70-3891-5945-98fb-dc099432e06a"
+
+[[Distributed]]
+deps = ["LinearAlgebra", "Random", "Serialization", "Sockets"]
+uuid = "8ba89e20-285c-5b6f-9357-94700520ee1b"
+
+[[DocStringExtensions]]
+git-tree-sha1 = "a016e0bfe98a748c4488e2248c2ef4c67d6fdd35"
+uuid = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
+version = "0.5.0"
+
+    [DocStringExtensions.deps]
+    LibGit2 = "76f85450-5226-5b5a-8eaa-529ad045b433"
+    Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
+    Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
+    Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+
+[[Documenter]]
+git-tree-sha1 = "9f2135e0e7ecb63f9c3ef73ea15a31d8cdb79bb7"
+uuid = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+version = "0.20.0"
+
+    [Documenter.deps]
+    Base64 = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
+    DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
+    InteractiveUtils = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
+    LibGit2 = "76f85450-5226-5b5a-8eaa-529ad045b433"
+    Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
+    Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
+    Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
+    REPL = "3fa0cd96-eef1-5676-8a61-b3b8758bbffb"
+    Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+    Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+    Unicode = "4ec0a83e-493e-50e2-b9ac-8f72acf5a8f5"
+
+[[InteractiveUtils]]
+deps = ["LinearAlgebra", "Markdown"]
+uuid = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
+
+[[LibGit2]]
+uuid = "76f85450-5226-5b5a-8eaa-529ad045b433"
+
+[[Libdl]]
+uuid = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+
+[[LinearAlgebra]]
+deps = ["Libdl"]
+uuid = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+
+[[Logging]]
+uuid = "56ddb016-857b-54e1-b83d-db4d58db5568"
+
+[[Markdown]]
+deps = ["Base64"]
+uuid = "d6f4376e-aef5-505a-96c1-9c027394607a"
+
+[[Pkg]]
+deps = ["Dates", "LibGit2", "Markdown", "Printf", "REPL", "Random", "SHA", "UUIDs"]
+uuid = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
+[[Pkg]]
```

Comment:
You seem to accidentally have two copies of `Pkg` in here.

Reply:
Well, one is the Pkg which `DocStringExtensions` depend on (the stdlib one) and the other one is the development version of Pkg so this is correct.

Reply:
I assume you'll change that before merging?

Reply:
Change what?

Reply:
How? The `DocStringExtension` deps is from the registry, and we can't change that, and we need a different uuid for `Pkg` to actually use the development version. I don't see the problem here.

Reply:
> I don't see the problem here.

Checking a bogus, wrong UUID into the git repo? They should both use the development version in that case and we should have a mechanism for replacing the sysimg copy.

Reply:
> in that case and we should have a mechanism for replacing the sysimg copy.

Yea that would be great, but we don't have this and hence we actively change the uuid of Pkg when testing/building docs: https://github.com/JuliaLang/Pkg.jl/blob/cb37ef1ad3d1eebc912b83ddcb414aa3d67631ca/.travis.yml#L34. This is nothing new introduced in this PR.

We are also no using DocStringExtensions in the Pkg docs, so it will not matter that there are different versions.